website/package.json

```
@@ -23,19 +23,19 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@docusaurus/core": "2.4.0",
-    "@docusaurus/plugin-client-redirects": "2.4.0",
-    "@docusaurus/preset-classic": "2.4.0",
+    "@docusaurus/core": "~2.4.0",
+    "@docusaurus/plugin-client-redirects": "~2.4.0",
+    "@docusaurus/preset-classic": "~2.4.0",
     "@mdx-js/react": "^1.6.21",
     "clsx": "^1.1.1",
     "medium-zoom": "^1.0.6",
     "prism-react-renderer": "^1.3.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "redocusaurus": "^1.0.1"
+    "redocusaurus": "~1.0.1"
```

```diff
-"redocusaurus": "^1.0.1"
+"redocusaurus": "~1.0.1"
```

現在、masterブランチでは、 `v1.6.0` が使用されています。

```
    "node_modules/redocusaurus": {
      "version": "1.6.0",
```

他のpackage強いと関連が無く、 `^` から `~` に変更するならば、`~1.6.0` 指定にするのが良いと思います。


```suggestion
    "redocusaurus": "~1.6.0"
```

---

確かに、、、

↓で対応しましたー
869193fe8bd58ad9cc5fe0d51a51833046724480